samples/cpp/benchmark_app/main.cpp

```diff
@@ -1094,9 +1094,9 @@ int main(int argc, char* argv[]) {
 
         if (!FLAGS_exec_graph_path.empty()) {
             try {
-                std::string fileName = fileNameNoExt(FLAGS_exec_graph_path);
-                ov::pass::Serialize serializer(fileName + ".xml", fileName + ".bin");
-                serializer.run_on_model(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()));
+                ov::serialize(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()),
+                              FLAGS_exec_graph_path,
+                              "");
```

Comment:
Is this planned? It seems like previously bin path was important as well.

Reply:
Yes, it's planned. Serialize ctor allows to pass bin_path as an empty string. In this case bin_path will be provided here
https://github.com/openvinotoolkit/openvino/blob/37e05afd1271e9ea22268af0cf25f98a26483474/src/core/src/pass/serialize.cpp#L969-L979